solidity/test/InterchainAccountRouter.t.sol

```diff
@@ -129,4 +129,30 @@ contract InterchainAccountRouterTest is Test {
         );
         assertEq(localIca.owner(), address(originRouter));
     }
+
+    function testReceiveValue(uint256 value) public {
+        vm.assume(value <= address(this).balance);
```

Comment:
Idk foundry enough, but will this test always have a balance?

Reply:
no

Reply:
I do :)